3scale/3scale-operator
# THREESCALE-11231 enable grafana v5 support

What
Enabling support of Grafana v5 to support enabling monitoring stack on 4.16 +

Verification
- Provision 4.15
- Install 3scale from this branch
- Follow monitoring documentation on this branch to configure monitoring with Grafana v4 & confirm it's functional
- Follow monitoring documentation on this branch on how to migrate to V5 & confirm it's functional
- Uninstall 3scale
- Install again with monitoring v4
- Upgrade OSD to 4.16
- Confirm v4 Grafana still works and migrate to v5 and confirm it works

TODO:
2 k8s dashboards are not working with v5 - doing investigation atm
